### PR TITLE
:arrow_up::one::one: pos_cashbox

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Comment by yelizariev** [comment]
For lints use :rainbow:

**Comment by yelizariev** [comment]
travis checks are too red

File: pos_cashbox/tests/test_js.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import odoo.tests
+
+
+@odoo.tests.common.at_install(False)
+@odoo.tests.common.post_install(True)
+class WebSuite(odoo.tests.HttpCase):
+
+    def test_01_js(self):
+        # webclient desktop test suite
+        self.phantom_js('/web/tests?mod=web&failfast', "", "", login='admin', timeout=1800)
```
Comment (yelizariev) [inline pos_cashbox/tests/test_js.py:10]: wrong

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_cashbox/tests/test_js.py (hunk 1 above)
Comment (GabbasovDinar) [reply pos_cashbox/tests/test_js.py:10]: image

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
:+1: "Open Cashbox" Button works in payment screen from POS

**Comment by GabbasovDinar** [comment]
` "Open Cashbox" Button works in payment screen from POS`

@Ramil-Mukhametzyanov It's a default feature. But this module adds a new button in form view for open cashbox

**Pull request closed without merging**